### DISCUSSION TITLE
How to install in cPanel file manager?

### DISCUSSION THREAD

**Discussion by Ordresser** [datenstrom/community#429] [Ask a question] [answered]
I unzipped Yellow in my domain in cPanel File Manager.

But when I go to mydomain.com, there's a 403 error there. When I go to mydomain.com/yellow-master, there is the CMS. 

How do I set the CMS (and the contents of what I blog there) to be at domain.com?

**Comment by schulle4u** [comment]
I'm not familiar with cPanel. However if your file manager supports file moving, enter the `yellow-master` subdirectory, select all files and folders (including .htaccess), and move them one directory level up. You can also download Yellow to your local computer, unzip it and upload the contents of `yellow-master` into your domain's root directory using FTP/SFTP.

**Comment by Ordresser** [comment]
Right, moving everything up into the main directory was the first thing I tried. The 403 persists. When I do domain.com/yellow.php I get a 'file not found' but it is using the CMS because it prints out 'Datenstrom yellow' and 'blog' 'about'

Does anything need to be configured in the files? Or do people just drop them all in the main directory (in my case it's home/myaccountname/public_html/domain.com/) and everything works without modifying any files?

(note: it's cPanel with apache through Godaddy)

**Comment by schulle4u** [comment, marked as answer]
It looks like your webhost doesn't parse the rewrite rules in Yellow's `.htaccess` file. Have you also moved it into the document root? Maybe your file manager missed it and needs to be configured to list hidden files.

**Comment by Ordresser** [comment]
I was wondering if this might have something to do with it. Actually, when I extract the package, the files that display in github aren't ALL there.
![Screenshot from 2019-05-22 16-00-49](https://user-images.githubusercontent.com/50929791/58212071-de74ec80-7caa-11e9-8afe-137e69e73d39.png)
There is no htaccess file in the package. Do I need to individually download all the files in github and upload them to my directory?
![Screenshot from 2019-05-22 16-03-42](https://user-images.githubusercontent.com/50929791/58212166-33186780-7cab-11e9-949a-2a9790be30fe.png)
When I unpack the folder I don't even get htaccess

(although I saved one and dropped just that one file in individually to test it)

**Comment by Ordresser** [comment]
UPDATE:

Yes, it was the htaccess file. 

Which was made invisible in settings by default in Godaddy's cPanel.

After making it visible, move all contents from yellow-master folder to root folder (domain.com in my case). Then refresh domain.com. It shows the setup screen for me.

Thanks for your help.
